### PR TITLE
[PfemFluid] cleaning

### PR DESCRIPTION
**Pull request by AFranci**
There are not singificant changes, just a general cleaning before the next more intrusive PRs.

### CODE REVIEW

File: applications/DelaunayMeshingApplication/custom_utilities/mesher_utilities.cpp [hunk 1]
```diff
@@ -395,308 +393,305 @@ namespace Kratos
...
-	array_1d<double, 3> Corner;
+      array_1d<double, 3> Corner;
 
-	for(unsigned int i = 0; i < size; ++i)
-	{
+      for (unsigned int i = 0; i < size; ++i)
+      {
 
-	    //std::cout<<" V: ("<<rGeometry[i].Id()<<"): ["<<rGeometry[i].X()<<", "<<rGeometry[i].Y()<<", "<<rGeometry[i].Z()<<"]  Normal:"<<rGeometry[i].FastGetSolutionStepValue(NORMAL)<<std::endl;
+        // std::cout<<" V: ("<<rGeometry[i].Id()<<"): ["<<rGeometry[i].X()<<", "<<rGeometry[i].Y()<<", "<<rGeometry[i].Z()<<"]  Normal:"<<rGeometry[i].FastGetSolutionStepValue(NORMAL)<<std::endl;
```
Comment (AlejandroCornejo) [inline applications/DelaunayMeshingApplication/custom_utilities/mesher_utilities.cpp]: not removing commented code?

File: applications/DelaunayMeshingApplication/custom_utilities/mesher_utilities.cpp [hunk 2]
```diff
@@ -395,308 +393,305 @@ namespace Kratos
...
+            if (projection < extra)
+              numextra++;
+          }
+        }
 
-	    double coplanar = inner_prod(Coplanar,Normal);
+        double coplanar = inner_prod(Coplanar, Normal);
 
-	    //std::cout<<" V["<<i<<"]: "<<rGeometry[i]<<" Normal:"<<Normal<<" coplanar "<<fabs(coplanar)<<" < "<<ortho<<std::endl;
+        // std::cout<<" V["<<i<<"]: "<<rGeometry[i]<<" Normal:"<<Normal<<" coplanar "<<fabs(coplanar)<<" < "<<ortho<<std::endl;
```
Comment (AlejandroCornejo) [inline applications/DelaunayMeshingApplication/custom_utilities/mesher_utilities.cpp]: idem

File: applications/DelaunayMeshingApplication/custom_utilities/mesher_utilities.cpp [hunk 3]
```diff
@@ -395,308 +393,305 @@ namespace Kratos
...
+        outer = false;
 
-	// if(numorthogonal>=1)
-	//   outer=false;
+      // if(numorthogonal>=1)
+      //   outer=false;
 
-	//std::cout<<std::endl;
-	//std::cout<<"  [ no:"<<numouter<<";ne:"<<numextra<<";nc:"<<numcoplanar<<";ns: "<<numsamedirection<<";nor:"<<numorthogonal<<"] ACCEPTED: "<<outer<<std::endl;
+      // std::cout<<std::endl;
```
Comment (AlejandroCornejo) [inline applications/DelaunayMeshingApplication/custom_utilities/mesher_utilities.cpp]: idem

File: applications/DelaunayMeshingApplication/custom_utilities/mesher_utilities.cpp [hunk 4]
```diff
@@ -838,608 +839,593 @@ namespace Kratos
...
-    double CriticalVolume =  1e-12 * pow( MinimumSideLength, size-1 );
+    double CriticalVolume = 1e-12 * pow(MinimumSideLength, size - 1);
 
-    //check sliver (volume)
-    if( Volume < CriticalVolume ){
-      //std::cout<<" Sliver (volume) "<<Volume<<" "<<CriticalVolume<<std::endl;
+    // check sliver (volume)
+    if (Volume < CriticalVolume)
+    {
+      // std::cout<<" Sliver (volume) "<<Volume<<" "<<CriticalVolume<<std::endl;
```
Comment (AlejandroCornejo) [inline applications/DelaunayMeshingApplication/custom_utilities/mesher_utilities.cpp:882]: kratos_warning ?

File: applications/DelaunayMeshingApplication/custom_utilities/mesher_utilities.cpp [hunk 5]
```diff
@@ -838,608 +839,593 @@ namespace Kratos
...
       }
-      else if( ContactType == EdgeToEdge ){ //EDGE_EDGE
+      else if (ContactType == EdgeToEdge)
+      { // EDGE_EDGE
 
-	//compare vertex normals (detect coplanar faces and orthogonal faces)
-	//if( !CheckVertexNormals(rGeometry) )
-	distorted = true;
+        // compare vertex normals (detect coplanar faces and orthogonal faces)
+        // if( !CheckVertexNormals(rGeometry) )
```
Comment (AlejandroCornejo) [inline applications/DelaunayMeshingApplication/custom_utilities/mesher_utilities.cpp]: kratos debug warning?

File: applications/DelaunayMeshingApplication/custom_utilities/mesher_utilities.cpp [hunk 6]
```diff
@@ -838,608 +839,593 @@ namespace Kratos
...
-	    if( i_nnode.Is(BOUNDARY) ){
-	      double x = i_nnode.X();
-	      double y = i_nnode.Y();
-	      double z = i_nnode.Z();
-	      double l = (x-xc)*(x-xc);
-	      l += (y-yc)*(y-yc);
-	      l += (z-zc)*(z-zc);
+    if ((BoundaryPoint.GetValue(NEIGHBOUR_NODES)).size() != 0)
+    {
+      double xc = BoundaryPoint.X();
```
Comment (AlejandroCornejo) [inline applications/DelaunayMeshingApplication/custom_utilities/mesher_utilities.cpp]: const?

File: applications/DelaunayMeshingApplication/custom_utilities/mesher_utilities.cpp [hunk 7]
```diff
@@ -1450,50 +1436,51 @@ namespace Kratos
...
+      array_1d<double, 3> P0;
+      noalias(P0) = rVertices[0].Coordinates() + MovementFactor * (rVertices[0].FastGetSolutionStepValue(DISPLACEMENT) - rVertices[0].FastGetSolutionStepValue(DISPLACEMENT, 1));
 
-      array_1d<double,3> P1;
-      noalias(P1) = rVertices[1].Coordinates() + MovementFactor * (rVertices[1].FastGetSolutionStepValue(DISPLACEMENT) - rVertices[1].FastGetSolutionStepValue(DISPLACEMENT,1));
+      array_1d<double, 3> P1;
+      noalias(P1) = rVertices[1].Coordinates() + MovementFactor * (rVertices[1].FastGetSolutionStepValue(DISPLACEMENT) - rVertices[1].FastGetSolutionStepValue(DISPLACEMENT, 1));
 
       double x10 = P1[0] - P0[0];
       double y10 = P1[1] - P0[1];
```
Comment (AlejandroCornejo) [inline applications/DelaunayMeshingApplication/custom_utilities/mesher_utilities.cpp]: const?

File: applications/DelaunayMeshingApplication/custom_utilities/mesher_utilities.cpp [hunk 8]
```diff
@@ -1508,401 +1495,375 @@ namespace Kratos
...
         }
       }
     }
-    double detF  = MathUtils<double>::Det(F);
+    double detF = MathUtils<double>::Det(F);
 
-    if(detF<0)
-      std::cout<<" NEGATIVE ELEMENT (DET_F: "<<detF<<")"<<std::endl;
+    if (detF < 0)
+      std::cout << " NEGATIVE ELEMENT (DET_F: " << detF << ")" << std::endl;
```
Comment (AlejandroCornejo) [inline applications/DelaunayMeshingApplication/custom_utilities/mesher_utilities.cpp]: kratos_warning_if

File: applications/PfemFluidDynamicsApplication/custom_processes/select_mesh_elements_for_fluids_process.hpp [hunk 9]
```diff
@@ -477,27 +471,27 @@ namespace Kratos
...
                         b2 = (nodesCoordinates[1][2] - nodesCoordinates[0][2]) * (nodesCoordinates[3][0] - nodesCoordinates[0][0]) - (nodesCoordinates[3][2] - nodesCoordinates[0][2]) * (nodesCoordinates[1][0] - nodesCoordinates[0][0]);
                         c2 = (nodesCoordinates[1][0] - nodesCoordinates[0][0]) * (nodesCoordinates[3][1] - nodesCoordinates[0][1]) - (nodesCoordinates[3][0] - nodesCoordinates[0][0]) * (nodesCoordinates[1][1] - nodesCoordinates[0][1]);
-                        double a3 = 0; //slope x for plane on the third triangular face of the tetrahedra (nodes B,C,D)
-                        double b3 = 0; //slope y for plane on the third triangular face of the tetrahedra (nodes B,C,D)
-                        double c3 = 0; //slope z for plane on the third triangular face of the tetrahedra (nodes B,C,D)
+                        double a3 = 0; // slope x for plane on the third triangular face of the tetrahedra (nodes B,C,D)
+                        double b3 = 0; // slope y for plane on the third triangular face of the tetrahedra (nodes B,C,D)
```
Comment (AlejandroCornejo) [inline applications/PfemFluidDynamicsApplication/custom_processes/select_mesh_elements_for_fluids_process.hpp]: const?

**Comment by AFranci** [comment]
thanks @AlejandroCornejo 
I avoided the warning on slivers because there would appear a lot...

**Review by AlejandroCornejo — APPROVED** [review]

**Pull request merged**